requirements-dev.txt

```diff
@@ -0,0 +1 @@
+black
```

Comment:
I think it's good to have a version used specified as well.

Reply:
I prefer to have always the latest black installed because we want to keep up with the latest formatting trends. If the build fails in the future because of black, just make quick changes.

Reply:
@tsudmi , I pinned black version in 193b08b as requested.